DESCRIPTION

```diff
@@ -4,11 +4,10 @@ Title: Material to support course on nowcasting and forecasting of infectious
 Version: 0.0.0.9999
 Authors@R: c(
     person(
-      given = "Sebastian",
-      family = "Funk",
-      role = c("cre", "cph", "aut"),
-      email = "sebastian.funk@lshtm.ac.uk",
-      comment = c(ORCID = "0000-0002-2842-3406")
+      given = "NFIDD course contributors",
+      family = "",
+      role = c("cre", "aut"),
+      email = "epiforecasts@gmail.com"
```

Comment:
do we need an email? I don't think anyone monitors that.

Reply:
just checked - 5 emails since 2022. Phew, nothing missed.

Reply:
I wasn't clear you could avoid having an email?